examples/components/mysql/service.jsonnet

```diff
@@ -0,0 +1,19 @@
+{
+  MySQLService(name): {
```

Comment:
why not use `kube.service` here to abstract a few parameters and show how you don't need to define the port twice (on statefulset and service)

Reply:
My idea was to show that you don't need to buy into Jsonnet, but a definition can be as simple as json.

You can then develop it and improve it over time. I wanted to show a more complete example (the deployment itself) and a simpler one.

Reply:
good idea. but I think we should make it clear that we're doing that. I think I lot of people go straight into the examples to learn how something works. we should probably put this into a subfolder called `examples/mysql_without_jsonnet` or something like that.